src/mdb_v8.c

```diff
@@ -3362,6 +3444,60 @@ jsobj_print_jsarray(uintptr_t addr, jsobj_print_t *jsop)
 }
 
 static int
+jsobj_print_jstyped_array(uintptr_t addr, jsobj_print_t *jsop)
```

Comment:
@davepacheco Sounds good, I hadn't seen your suggestion of "\<TypedArray of length N\>" initially, because to display a string within "\<\>" in GitHub comments, you need to escape "\<" and "\>" apparently. It definitely helps me to understand what you meant :) I updated your original comment, which is why we can see it now.

To summarize, I will just make `::jsprint` display "\<TypedArray of length N\>" for typed arrays.